module_4/framework/public/index.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Framework\Core\Request;
+use Framework\Core\Response;
+use Framework\Core\Router;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+$container = new ContainerBuilder();
+
+
+function sendResponse(\Psr\Http\Message\ResponseInterface $response)
+{
+
+    http_response_code($response->getStatusCode());
+
+    foreach ($response->getHeaders() as $name => $values) {
+        foreach ($values as $value) {
+            header(sprintf('%s: %s', $name, $value), false);
+        }
+    }
+
+    echo $response->getBody();
+}
+
+
+$router = new Router();
+$router->register('GET', '/', function($request) {
+    $controller = new \Framework\Controller\HomeController();
+    return $controller->index($request);
```

Comment:
```suggestion

    return $controller->index($request);
```
@Hlopec It's a best practice.